app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/huami/miband3/MiBand3Support.java

```diff
@@ -71,6 +75,45 @@ protected AmazfitBipSupport setDisplayItems(TransactionBuilder builder) {
         return this;
     }
 
+    @Override
+    public void onSendConfiguration(String config) {
+        TransactionBuilder builder;
+        try {
+            builder = performInitialized("Sending configuration for option: " + config);
```

Comment:
I am just unsure about overriding onSendConfiguration, because performInitialized may be called twice in very quick succession. Can that be a problem?

Reply:
In theory, that shouldn't be a problem :grimacing: 
Both calls will just add actions to the queue. The queue is properly synchronized. Initialization (if necessary) will only happen once.

Reply:
Alright, that was my concern. Then everything should be okay.